Use gcc for linking because of -Wl,... from "perl -MExtUtils::Embed -e ldopts".

--- src/config/Linux_All.mk.orig	2003-05-16 02:21:08.000000000 +0200
+++ src/config/Linux_All.mk	2004-03-02 03:56:15.000000000 +0100
@@ -42,7 +42,7 @@
 OS_CFLAGS = -DXP_UNIX -DSVR4 -DSYSV -D_BSD_SOURCE -DPOSIX_SOURCE -DHAVE_LOCALTIME_R
 
 RANLIB = echo
-MKSHLIB = $(LD) -shared $(XMKSHLIBOPTS)
+MKSHLIB = $(CC) -shared -lc -Wl,-soname,libjs.so.1 $(XMKSHLIBOPTS)
 
 #.c.o:
 #      $(CC) -c -MD $*.d $(CFLAGS) $<
